demo-app/src/zcashtestnet/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="app_name">Testnet Demo</string>
     <string name="nav_header_title">Android SDK Demo : TESTNET</string>
+    <string name="network_name">Testnet</string>
```

Comment:
Out of curiosity, how does the user select mainnet or testnet? is it a build flavor / target or a dynamic setting?

Reply:
The build flavor selects it automatically, as there's a resource file under both `demo-app/src/zcashtestnet/res/values/strings.xml` and `demo-app/src/zcashmainnet/res/values/strings.xml`